yosoi/cli/main.py

```
@@ -34,6 +34,21 @@ def _resolve_output_formats(flag_values: tuple[str, ...]) -> list[str]:
+def _collect_urls(url: str | None, file_path: str | None, limit: int | None) -> list[str]:
+    """Build the URL list from --url / --file / --limit flags."""
+    urls: list[str] = []
+    if url:
+        urls.append(url)
+    if file_path:
+        urls.extend(load_urls_from_file(file_path))
+    if not urls:
+        raise click.UsageError('No URLs provided. Use --url <url> or --file <file>')
+    if limit:
+        urls = urls[:limit]
+        console.print(f'[cyan]ℹ Limiting to first {limit} URLs[/cyan]')
```

Ruff (0.15.12)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)